src/components/Footer/Footer.jsx

```diff
@@ -4,15 +4,126 @@ import React from 'react';
+const RenderCallToActionSection = ({ isReallySmallScreen }) => (
+  <Stack width={isReallySmallScreen ? 1 : 3 / 5} alignItems="center" justifyContent="center">
+    <Typography variant="h5" color="tertiary.main">
+      Want to partner with PASS?
+    </Typography>
+    <Typography variant="body1" color="#fff" sx={{ width: 3 / 4 }}>
+      If your organization is interested in partnering with PASS and would like to discuss further,
+      contact us below.
+    </Typography>
+    <Button variant="contained" color="secondary" sx={{ my: '1rem', width: 1 / 2 }}>
+      Proposal
+    </Button>
+  </Stack>
+);
+
+const socialLinks = [
+  {
+    href: 'https://twitter.com/',
+    icon: <TwitterIcon />
+  },
+  {
+    href: 'https://www.facebook.com/',
+    icon: <FacebookIcon />
+  },
+  {
+    href: 'https://www.instagram.com/',
+    icon: <InstagramIcon />
+  }
+];
+
+// middle section of footer
+const RenderCompanyInfoSection = ({ isReallySmallScreen }) => (
+  <Stack
+    width={isReallySmallScreen ? 1 : 1 / 5}
+    spacing={2}
+    justifyContent="space-around"
+    alignItems="center"
+  >
+    <Box>
+      <Typography color="tertiary.main">PASS LOGO</Typography>
+      <Typography color="#fff">tagline</Typography>
+    </Box>
+    <Box>
+      <Typography color="tertiary.main">Follow Us</Typography>
+      <Stack direction="row" spacing={1}>
+        {socialLinks.map(({ href, icon }) => (
+          <Link key={href} href={href} target="_blank" rel="noopener" color="#fff">
+            {icon}
+          </Link>
+        ))}
+      </Stack>
+    </Box>
+    <Box>
+      <Typography color="tertiary.main">Built By:</Typography>
+      <Link href="https://www.codeforpdx.org/" target="_blank" rel="noopener">
+        <Typography variant="body2" color="#fff">
+          C4PDX LOGO
+        </Typography>
+      </Link>
+    </Box>
+  </Stack>
+);
+
+const legalLinks = [
+  {
+    href: 'https://www.codeforpdx.org/',
+    title: 'Privacy Policy'
+  },
+  {
+    href: 'https://www.codeforpdx.org/',
+    title: 'Terms and Conditions'
+  },
+  {
+    href: 'https://www.codeforpdx.org/',
+    target: '_blank',
+    rel: 'noopenner',
+    ml: 0.5,
+    text: `©${new Date().getFullYear()}`,
+    title: 'Code for PDX'
+  }
+];
+
+// bottom section of footer
+const RenderCopyrightAndLinksSection = ({ isReallySmallScreen }) => (
+  <Stack
+    orientation="column"
+    width={isReallySmallScreen ? 1 : 1 / 5}
+    spacing={2}
+    justifyContent="space-around"
+  >
+    {legalLinks.map((link) => (
+      <Typography key={link.title} variant="body2" color="tertiary.main">
+        {link.text ?? null}
+        <Link
+          href={link.href}
+          underline="none"
+          color="tertiary.main"
+          target={link.target ?? null}
+          rel={link.rel ?? null}
+          ml={link.ml ?? null}
+        >
+          {link.title}
+        </Link>
+      </Typography>
+    ))}
+  </Stack>
+);
+
 /**
  * Footer Component - Footer Component for PASS
```

Comment:
not sure if this jsdoc is ideal...  maybe more to it?

Reply:
I'm thinking about elaborating on it a bit more like: `Footer Component - Generates the responsive footer used in PASS` but it does look a bit bare.

What do you have in mind?

Reply:
Not sure. I’m still terrible with Jason, but I feel like most other components had a “member of ……” or something else at minimum.

Reply:
Oh! Do you mean for the internal components in the Footer? Yeah, we can certainly do that. I usually just limit to components directly being used on the  pages, but there's nothing stopping us from doing it.

Do you want it included?

Reply:
Nah, just a better jsdoc comment was fine.    I'm still lost on jsdoc, I will research it more and watch some videos on it soon.